chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SchedulerController.java

```diff
@@ -96,10 +105,11 @@ public void deleteTrigger(@PathVariable("triggerGroup") final String triggerGrou
             notes = "Security Restrictions: ROLE_ADMIN, ROLE_ONC, or ROLE_ACB and have administrative "
                     + "authority on the specified ACB.")
     @RequestMapping(value = "/triggers", method = RequestMethod.GET, produces = "application/json; charset=utf-8")
-    public @ResponseBody ScheduleTriggersResults getAllTriggers() throws SchedulerException {
+    public @ResponseBody ScheduleTriggersResults getAllTriggers(final HttpServletResponse response) throws SchedulerException {
```

Comment:
Why was the response variable added? The diff doesn't show it being used in the method.

Reply:
Good catch - that was left from a previous attempt at a fix and should have been removed.  I'll take remove of it.